### ISSUE TITLE
Tables should be navigated via arrow keys

### ISSUE THREAD

**Issue by TimoSto** [TimoSto/ThesorTeX#223]
- add a caption to the table as a whole
- table as a whole can be reached using tab
- rows can be reached using arrow keys
- buttons can still be reached using tab
